packages/g/google-libphonenumber.json

```
@@ -0,0 +1,41 @@
+{
+  "description": "The up-to-date and reliable Google's libphonenumber package for node.js for parsing, formatting, and validating international phone numbers.",
+  "filename": "dist/libphonenumber.js",
```

```suggestion
  "filename": "libphonenumber.js",
```
As the base dir is `dist`, we'll pull the files out of that directory into the root, so it isn't needed here.